nservicebus/upgrades/5to6-recoverability.md

```diff
@@ -72,4 +72,14 @@ snippet: 5to6-configureImmediateRetriesViaXml
 
 snippet: 5to6-configureDelayedRetriesViaCode
 
-snippet: 5to6-configureDelayedRetriesViaXml
\ No newline at end of file
+snippet: 5to6-configureDelayedRetriesViaXml
+
+## Legacy .Retries queue
```

Comment:
@indualagarsamy I like it :+1: . I've incorporated it with minor changes to upgrade guide.